services/ark-api/ark-api/src/ark_api/api/v1/openai.py

```diff
@@ -54,6 +57,62 @@ class ChatCompletionRequest(BaseModel):
     stream: bool = False
 
 
+async def try_get_streaming_url(ark_client, query_name: str, namespace: str = "default") -> Optional[str]:
```

Comment:
this needs to be clearer in the comments. It is used to get the canonical streaming address for a query. It is possible to open a connection to this endpoint in advance of a query, during a query, or after a query has completed.

Reply:
no `default` lets be explicit